Databricks/Alluxio-2.9.0-without-CVE/patches/readme.txt

```diff
@@ -0,0 +1 @@
+The patch files is from this PR: https://github.com/res-life/alluxio/pull/1
```

Comment:
we dont' need this if patch files in this pr.

Reply:
Removed this `readme` file.